cns/NetworkContainerContract.go

```diff
@@ -78,10 +78,18 @@ type NICType string
 // NIC Types
 const (
 	InfraNIC NICType = "InfraNIC"
-	// Delegated VM NICs are projected from VM to container network namespace
-	DelegatedVMNIC NICType = "DelegatedVMNIC"
-	// BackendNIC NICs are used for infiniband nics on a VM
+	// DelegatedVMNIC are projected from VM to container network namespace
+	DelegatedVMNIC NICType = "FrontendNIC"
+	// BackendNIC are used for infiniband NICs on a VM
 	BackendNIC NICType = "BackendNIC"
```

Comment:
do we need to keep this BackendNIC nictype var for any reason similar to the DelegatedVMNIC variable? (like DNC is referring to this value?)